[ZT] RBI User Action logs (#20635)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/policies/browser-isolation/setup/clientless-browser-isolation.mdx

- **Authentication events**: User login events are available in [Access audit logs](/cloudflare-one/insights/logs/audit-logs/).
- **HTTP requests**: Traffic from the remote browser to the Internet is logged in [Gateway activity logs](/cloudflare-one/insights/logs/gateway-logs/).
- **DNS queries**: DNS queries from the remote browser are shown in [Gateway activity logs](/cloudflare-one/insights/logs/gateway-logs/).

> - **User actions**: Track how often users perform copy/paste, download/upload, and print actions in the remote browser and on which websites. (Only available in [Logpush](/cloudflare-one/insights/logs/logpush/).)

Reviewer: ```suggestion
- **User actions**: Track how often users perform copy/paste, download/upload, and print actions per each website and in the remote browser (only available in [Logpush](/cloudflare-one/insights/logs/logpush/)).
```
